docs/source/getting-started/what-is-jupyterhub.md

```diff
@@ -0,0 +1,465 @@
+and can be used to discover most problems.
+
+The main pieces of JupyterHub are:
+
+### Authenticator
+
+JupyterHub itself doesn't actually manage your users.  It has a
+database of users, but it is usually connected with some other system
+that manages the usernames and passwords.  When someone tries to log
+in to JupyteHub, it just asks the
+**authenticator**([basics](authenticators-users-basics),
+[reference](../reference/authenticators)) if the
+username/password is valid(&).  The authenticator can also return user
+groups and admin status of users, so that JupyterHub can do some
+higher-level management.  The authenticator returns a username(&),
+which is passed on to the spawner, which has to use it to start that
+user's environment.
+
+The following authenticators are included with JupyterHub:
+
+- **PAMAuthenticator** uses the standard Unix/Linux operating system
+  functions to check users.  Roughly, if someone already has access to
+  the machine (they can log in by ssh), they will be able to log in to
+  JupyterHub without any other setup.  Thus, JupyterHub fills the role
+  of a ssh server, but providing a web-browser based way to access the
+  machine.
+
+
+But those are fairly limited, and thus there are [plenty of others to
+choose
+from](https://github.com/jupyterhub/jupyterhub/wiki/Authenticators).
+You can connect to almost any other existing service to manage your
+users.  You either use all users from this other service (e.g. your
+company), or whitelist only the allowed users (e.g. your group's
+Github usernames).  Some other popular authenticators include:
+
+- **OAuthenticator** uses the standard OAuth protocol to verify users.
+  For example, you can easily use Github to authenticate your users -
+  people have a "click to login with Github" button.  This is often
+  done with a whitelist to only allow certain users.
+
+- **NativeAuthenticator** actually stores and validates its own
+  usernames and passwords, unlike most other authenticators.  Thus,
+  you can manage all your users within JupyterHub only.
+
+- There are authenticators for LTI (learning management systems),
+  Shibboleth, Kerberos - and so on.
+
+The authenticator is configured with the
+`c.JupyterHub.authenticator_class` configuration option in the
+`jupyterhub_config.py` file.
+
+The authenticator runs internally to the Hub process but communicates
+with outside services.
+
+If you have trouble logging in, this is usually a problem of the
+authenticator.  The authenticator logs are part of the the JupyterHub
+logs, but there may also be relevant information in whatever external
+services you are using.
+
+### Spawner
+
+The **spawner** ([basics](spawners-basics),
+[reference](../reference/spawners)) is the real core of
+JupyterHub: when someone wants a notebook server, it allocates
+resources and starts the server.  The notebook server could run on the
+same server as JupyterHub, on another server, on some cloud service,
```

Comment:
```suggestion
same machine as JupyterHub, on another machine, on some cloud service,
```
I think using a different word to make a distinction between "server" (the process) and "server" (the physical device) would be helpful. Not sure if "machine" or "node" is the better choice or maybe "computer"?